packages/eslint-config-react-app/base.js

```diff
@@ -43,5 +43,6 @@ module.exports = {
   rules: {
     'react/jsx-uses-react': 'warn',
     'react/jsx-uses-vars': 'warn',
+    'react/react-in-jsx-scope': 'error',
```

Comment:
Are we still able to override this rule if it's in the base config?

Reply:
Yes, correct. I tested this locally.